.circleci/config.yml

```diff
@@ -932,6 +957,9 @@ jobs:
       name: win/default
       shell: cmd.exe
     steps:
+      # NOTE: For bytecode generation we need the input files to be byte-for-byte identical on all
+      # platforms so line ending conversions must absolutely be disabled.
+      - run: git config --global core.autocrlf false
```

Comment:
We could put this in `.gitattributes` but then it would also change for the build job and for contributors who work on Windows. I think that for the build job either is fine (with it enabled we're getting extra testing for CRLF versions) but not for users - we'd probably start getting CRLF endings in PRs. So I'm changing this only for specific jobs.